includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -339,6 +341,7 @@ class AMP_Allowed_Tags_Generated {
 			'hr',
 			'i',
 			'image',
+			'img',
```

Comment:
Also apparently not allowed yet: https://github.com/ampproject/amphtml/blob/4ce3cd79520dbeaf5ed5364cbff58d3d71dee40e/extensions/amp-story/validator-amp-story.protoascii#L712-L868

Reply:
Humm seems like it's only allowed for transformed AMP in most of the extensions. In that case, I have to add a condition to only add `img` to allowed extensions.